frontend/templates/get_involved.html

```diff
@@ -67,8 +67,8 @@ <h2>How can I get involved?</h2>
     <ul>
       <li>Contact your local laboratory team, CCG or CSU to encourage them to support the project by becoming an early partner.</li>
       <li>Help us design amazing tools. On the <a href="/">home page</a> you can see a tiny preview and read some background studies; our prescribing tool <a href="https://openprescribing.net">OpenPrescribing.net</a> gives a working demonstration of how much we can do with healthcare data. <a href="mailto:ebmdatalab@phc.ox.ac.uk">Let us know</a>what you would hope to get out of a pathology feedback tool as a practice, or anything specific you'd like it to include. We will share the pilot tool for your feedback as soon as possible.</li>
-      <li>Stay in touch! Keep an eye on this website for updates, and <a href="mailto:ebmdatalab@phc.ox.ac.uk">send us an email</a> if you would like to be added to our OpenPathology mailing list to which we will send occasional updates and ask for input/feedback.</li>
-      <li>Hopefully you are already making use of our <a href="https://openprescribing.net/">OpenPrescribing</a> service, including our email alerts service customised for your practice. We send a monthly OpenPrescribing newsletter in which we will mention any major OpenPathology news. You can sign up for this on the <a href="https://openprescribing.net">/OpenPrescribing.net home page</a>.</li>
+      <li>Stay in touch! Keep an eye on this website for updates, <a href="mailto:ebmdatalab@phc.ox.ac.uk">get in touch</a> with your thoughts/ideas or <a href="eepurl.com/gA3peX">sign up</a> to our OpenPathology mailing list to which we will send occasional updates and ask for input/feedback.</li>
+      <li>Hopefully you are already making use of our <a href="https://openprescribing.net/">OpenPrescribing</a> service, including our email alerts service customised for your <a href="https://openprescribing.net/practice">practice</a>.</li>
```

Comment:
add "or CCG"?

Reply:
this is the page "for GPs" so I just kept it as it was but we can add that if you like? No strong feeling either way

Reply:
I think people other than GPs will end up there, and only linking to practice pages means we might miss out on a couple more signups (agree it's not super important but that's my feeling..)